Makefile

```diff
@@ -17,7 +17,7 @@ codegen:
 .PHONY: lint
 lint:
 	docker run --rm -v "$(shell pwd):/var/work:ro" -w /var/work \
-		golangci/golangci-lint:v1.44.0 golangci-lint run -v --timeout=5m
+		golangci/golangci-lint:v1.46.0 golangci-lint run -v --timeout=5m
```

Comment:
If we're updating golangci-lint (why are we doing this in this PR?) then maybe bump it all the way to 1.55.2 or whatever is the latest.

Reply:
This is not an intended change for this PR but the upgrade of linodego lead causing https://github.com/golangci/golangci-lint-action/issues/448 issue on linodego package. I had to upgrade it to latest compatible version for it to work. 

I already tried upgrading it to the latest version (1.55.2). In this case, it's flagging multiple issues in the existing code (not the one being edited on the PR). That's the reason, I have upgraded it to the lowest compatible version.

I guess upgrading it to 1.55.2 might require a PR of it's own.